lib/mindee/parsing/custom/list_field.rb

```diff
@@ -14,13 +14,16 @@ class ListFieldItem
         attr_reader :polygon
         # @return [Array, Hash, String, nil]
         attr_reader :content
+        # @return [Integer, nil]
+        attr_reader :page_id
 
         # @param prediction [Hash]
-        def initialize(prediction)
+        def initialize(prediction, page_id: nil)
           @content = prediction['content']
           @confidence = prediction['confidence']
           @polygon = Geometry.polygon_from_prediction(prediction['polygon'])
           @bounding_box = Geometry.get_bounding_box(@polygon) unless @polygon.nil? || @polygon.empty?
+          @page_id = page_id
```

Comment:
`page_id` should be obtained from prediction

Reply:
Yeah but this allows to keep `page_id`s when they come from `pages`. Isn't it worth keeping?